src/webhookServer/handcashWebhookAuthentication.ts

```diff
@@ -0,0 +1,26 @@
+import express from 'express';
+import dotenv from "dotenv";
+import crypto from 'crypto';
+import {Types} from "@handcash/handcash-connect";
+
+dotenv.config();
+const appSecret = process.env.HANDCASH_APP_SECRET || '';
+
+
+const handcashWebhookAuthentication = (req: express.Request, res: express.Response, next: express.NextFunction) => {
+    const signature = req.headers['handcash-signature'] as string;
+    const body: Types.WebhookPayload = req.body;
+    const fiveMinutesAgo = new Date(new Date().getTime() - (5 * 60000));
+    if (new Date(body.created) < fiveMinutesAgo) {
+        return res.status(401).send('Timestamp is too old');
+    }
+    const hmac = crypto.createHmac('sha256', appSecret);
```

Comment:
The cryptographic checks could be covered by the SDK directly to avoid redundancy and encourage good security practices. Similar to what we discussed from Stripe.